.travis.yml

```diff
@@ -0,0 +1,23 @@
+dist: focal
+
+language: c
+
+sudo: required
+
+os: linux
+
+env:
+    - LUA_VERSION=5.1
+    - LUA_VERSION=5.2
+    - LUA_VERSION=5.3
```

Comment:
5.4 is also released...

Reply:
I know. Will add that later, since there is no Ubuntu package yet and `mockagne` needs a fix in order to be compatible with 5.3.